serverless.yml

```diff
@@ -107,23 +163,18 @@ resources:
           Ref: BertlyVPC
         SecurityGroupIngress:
         - IpProtocol: tcp
+          Description: 'Allow access to RDS Postgres database.'
           FromPort: '5432'
           ToPort: '5432'
           SourceSecurityGroupId:
             Ref: BertlySecurityGroup
-        - IpProtocol: tcp
-          FromPort: '6379'
-          ToPort: '6379'
-          SourceSecurityGroupId:
-            Ref: BertlySecurityGroup
     BertlyRDSSubnetGroup:
       Type: AWS::RDS::DBSubnetGroup
       Properties:
         DBSubnetGroupDescription: 'RDS Subnet Group'
         SubnetIds:
-        - Ref: BertlySubnetA
-        - Ref: BertlySubnetB
-        - Ref: BertlySubnetC
+        - Ref: BertlyPrivateSubnetA
+        - Ref: BertlyPrivateSubnetB
```

Comment:
Ah, so here's my actual subnet question. Why are there two private subnets?

Reply:
Amazon requires that RDS instances [have a subnet group with at least two subnets in two different availability zones](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbsubnet-group.html). My guess is that's the minimum for them to ensure their availability guarantees?

Reply:
Got it! 👍